src/manifolds/HyperbolicHyperboloid.jl

```diff
@@ -239,6 +239,120 @@ function exp!(M::Hyperbolic, q, p, X)
     return copyto!(q, cosh(vn) * p + sinh(vn) / vn * X)
 end
 
+function get_basis(M::Hyperbolic, p, B::DefaultOrthonormalBasis)
+    n = manifold_dimension(M)
+    V = [_hyperbolize(M, p, [i == k ? 1 : 0 for k in 1:n]) for i in 1:n]
+    _gram_schmidt!(M, V, p, V)
+    return CachedBasis(B, V)
+end
+
+function get_basis(M::Hyperbolic, p, B::DiagonalizingOrthonormalBasis)
+    n = manifold_dimension(M)
+    X = B.frame_direction
+    V = [_hyperbolize(M, p, [i == k ? 1 : 0 for k in 1:n]) for i in 1:n]
+    κ = -ones(n)
+    if norm(M, p, X) != 0
+        placed = false
+        for i in 1:n
+            if norm(M, p, V[i] - inner(M, p, X, V[i]) .* V[i]) ≈ 0 # is X a multiple of V[i]?
```

Comment:
Wouldn't `inner(M, p, X, V[i]) ≈ 1` be equivalent?

Reply:
I hope not, since I try to check whether V[i] is parallel to X, to avoid that the default placement (V[1|=X) does not yield a linear dependence.

Reply:
Then I think `inner(M, p, X, V[i]) ≈ norm(M, p, X)` looks correct. Or maybe `norm(M, p, X[i] - inner(M, p, X, V[i]) .* V[i]) ≈ 0` but the first variant look cleaner to me.

Reply:
Is `X[i]` a `V[i]` in your second formula? But you're right, the first looks cleaner and should also work.

Reply:
Hm, it was actually supposed to be just `X`. For example `[2, 0] - dot([2, 0], [1, 0]) .* [1, 0]` is approximately 0.

Reply:
Right, that should also work, but the norm line is nicer, will add that tomorrow.

Reply:
Oh, I think your inner line is missing an abs on the left to account for opposing directions (then inner would be minus the norm)

Reply:
Yes, there should be `abs`. It's good that you noticed.